pymaker/deployment.py

```diff
@@ -216,7 +216,7 @@ def from_json(web3: Web3, conf: str):
                 else:
                     gem = DSToken(web3, Address(conf[name[1]]))
 
-                if name[1] in ['USDC', 'WBTC']:
+                if name[1] in ['USDC', 'WBTC', 'TUSD']:
```

Comment:
Not sure if it matters here, but TUSD is closer to `GemJoin` than it is to `GemJoin5` since TUSD has 18 decimals.

Reply:
Which is fine; I tested and confirmed `dec` returns 18.